src/i18n/en-US/index.ts

```diff
@@ -786,11 +786,12 @@ export default {
     add: 'Add',
     unbond: 'Unbond',
     unlock: 'Unlock',
+    unlocking: 'Unlocking',
     stake: 'Stake',
     move: 'Move',
     index: 'Index',
     chunk: 'Chunk',
-    unbondingAmount: 'Unbonding amount',
+    unlockingAmount: 'unlocking amount',
```

Comment:
Maybe to change to 'Unlocking amount'. 
<img width="201" alt="image" src="https://github.com/AstarNetwork/astar-apps/assets/8452361/3af96241-fcb6-49ac-8321-cf4f71884666">

Reply:
@bobo-k2 Thank you! fixed in [cb97b53](https://github.com/AstarNetwork/astar-apps/pull/1148/commits/cb97b5397e81eb283089e10d33f1a2d046928b20)